4.9/ga/troubleshooting.md

```diff
@@ -266,19 +267,14 @@ Failures here usually point to an incomplete Checklist item.  Please login to co
 ## <a id="merge-pr"></a>merge-pr
 Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
 
-
-
-
-
-
-
-
-
+## <a id="verify-project-distribution"></a>verify-project-distribution
+Failures here means there is a mismatch between the destination catalog (e.g "Red Hat Certified", "Red Hat Marketplace") selected during project setup, and the GitHub repository against which the pull request was raised.
```

Comment:
Failures in this step suggest a mismatch between the destination catalog (e.g "Red Hat Certified", "Red Hat Marketplace") selected during project setup on connect.redhat.com and your pull request's GitHub repository.

Reply:
Was going to say may be it would be a good idea to link to or mention that config that needs to specified for marketplace bundles but unsure if that matters ..